data/src/main/java/org/oppia/data/backends/gae/NetworkModule.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.data.backends.gae
+
+import dagger.Module
+import dagger.Provides
+import okhttp3.OkHttpClient
+import org.oppia.data.backends.gae.api.ClassroomService
+import org.oppia.data.backends.gae.api.TopicService
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+/**
+ * Sample resource https://github.com/gahfy/Feed-Me/tree/unitTests
+ */
+
+/**
+ * Module which provides all required dependencies about network
+ */
+@Module
+// Safe here as we are dealing with a Dagger 2 module
+@Suppress("unused")
```

Comment:
This actually isn't needed since the warning is incorrect. The module is being used (and Android Studio will stop complaining about the lack of use after build files are updated).

Reply:
Removed. @Suppress